libs/SmartSync/SmartSync/Classes/Util/SFSyncOptions.h

```diff
@@ -44,11 +40,6 @@ extern NSString * const kSFSyncOptionsMergeMode;
 + (SFSyncOptions*) newSyncOptionsForSyncUp:(NSArray*)fieldlist;
 + (SFSyncOptions*) newSyncOptionsForSyncUp:(NSArray*)fieldlist mergeMode:(SFSyncStateMergeMode)mergeMode;
 
-+ (SFSyncOptions*) newSyncOptionsForSyncUp:(NSArray*)fieldlist
```

Comment:
Again, create and update field list were introduced after 5.0 shipped. So this is not a API change.